### DISCUSSION TITLE
How will the new portable (pooling) format affect chiadog?

### DISCUSSION THREAD

**Discussion by kanasite** [martomi/chiadog#250] [Q&A] [answered]
Will partial proof keep triggering the notification? Just wonder if chiadog will be supporting the new format and what are the changes will be

**Comment by martomi** [comment, marked as answer]
I think option to disable these notifications will make most sense.

**Reply by BigPeteL** [reply]
I'm seeing this too, also that as i'm not solo farming I get a Harvester alert, even though the pool farming is running ok. @martomi do you think this will be looked at post the official release of the pool-puzzles branch?

**Reply by kanasite** [reply]
perhaps change how "proof" is being parsed for the notification? 

for solo: potentially what's more important will be knowing if there's a block farmed "Made the unfinished block" "🍀 ️Farmed unfinished_block"

vs pool farming: "Submitting partial for"

**Reply by martomi** [reply]
There's now also an issue, lets consolidate the discussions to there: https://github.com/martomi/chiadog/issues/252
